bigtable/filter_samples.rb

```diff
@@ -0,0 +1,252 @@
+  read_with_filter project_id, instance_id, table_id, filter
+  # [END bigtable_filters_limit_col_range]
+end
+
+def filter_limit_value_range project_id, instance_id, table_id
+  # [START bigtable_filters_limit_value_range]
+  range = Google::Cloud::Bigtable::ValueRange.new.from("PQ2A.190405").to("PQ2A.190406")
+  filter = Google::Cloud::Bigtable::RowFilter.value_range range
+  read_with_filter project_id, instance_id, table_id, filter
+  # [END bigtable_filters_limit_value_range]
+end
+
+def filter_limit_value_regex project_id, instance_id, table_id
+  # [START bigtable_filters_limit_value_regex]
+  filter = Google::Cloud::Bigtable::RowFilter.value "PQ2A.*$"
+  read_with_filter project_id, instance_id, table_id, filter
+  # [END bigtable_filters_limit_value_regex]
+end
+
+def filter_limit_timestamp_range project_id, instance_id, table_id
+  # [START bigtable_filters_limit_timestamp_range]
+  timestamp_minus_hr = (Time.now.to_f * 1_000_000).round(-3) - 60 * 60 * 1000 * 1000
+  puts timestamp_minus_hr
+  filter = Google::Cloud::Bigtable::RowFilter.timestamp_range from: 0, to: timestamp_minus_hr
+
+  read_with_filter project_id, instance_id, table_id, filter
+  # [END bigtable_filters_limit_timestamp_range]
+end
+
+def filter_limit_block_all project_id, instance_id, table_id
+  # [START bigtable_filters_limit_block_all]
+  filter = Google::Cloud::Bigtable::RowFilter.block
+  read_with_filter project_id, instance_id, table_id, filter
+  # [END bigtable_filters_limit_block_all]
+end
+
+def filter_limit_pass_all project_id, instance_id, table_id
+  # [START bigtable_filters_limit_pass_all]
+  filter = Google::Cloud::Bigtable::RowFilter.pass
+  read_with_filter project_id, instance_id, table_id, filter
+  # [END bigtable_filters_limit_pass_all]
+end
+
+def filter_modify_strip_value project_id, instance_id, table_id
+  # [START bigtable_filters_modify_strip_value]
+  filter = Google::Cloud::Bigtable::RowFilter.strip_value
+  read_with_filter project_id, instance_id, table_id, filter
+  # [END bigtable_filters_modify_strip_value]
+end
+
+def filter_modify_apply_label project_id, instance_id, table_id
+  # [START bigtable_filters_modify_apply_label]
+  filter = Google::Cloud::Bigtable::RowFilter.label "labelled"
+  read_with_filter project_id, instance_id, table_id, filter
+  # [END bigtable_filters_modify_apply_label]
+end
+
+def filter_composing_chain project_id, instance_id, table_id
+  # [START bigtable_filters_composing_chain]
+  filter = Google::Cloud::Bigtable::RowFilter.chain.cells_per_column(1).family("cell_plan")
+  read_with_filter project_id, instance_id, table_id, filter
+  # [END bigtable_filters_composing_chain]
+end
+
+def filter_composing_interleave project_id, instance_id, table_id
+  filter = Google::Cloud::Bigtable::RowFilter.interleave.value("true").qualifier("os_build")
+  read_with_filter project_id, instance_id, table_id, filter
+  # [END bigtable_filters_composing_interleave]
```

Comment:
Either this is missing the "START" tag or I am

Reply:
fixed